src/main.rs

```diff
@@ -118,10 +135,78 @@ fn consume_emitter_err(res: emitters::EmitterResult, filepath: &str) {
     }
 }
 
+fn consume_edit_script_err(error: &ast::ArenaError) -> ! {
```

Comment:
Oddly, the compiler could deduce the ! on `consume_emitter_err` but not this function.